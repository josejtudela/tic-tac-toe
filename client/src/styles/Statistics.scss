.girdStatistics {
    width: 60vw;
    display: grid;
    grid-template-columns: repeat(2, 1fr);
    
    .girdReact {
        border-top: 5px solid #00d8ff;
    }
    .girdVue {
        border-top: 5px solid #42b883;
    }
}

.gridMovements {
    display: grid;
    grid-template-columns: 10% 10% 10%;
    row-gap: 0.5em;
}