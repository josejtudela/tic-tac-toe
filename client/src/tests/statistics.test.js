import React from 'react';
import { shallow, mount, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import axios from 'axios';
import Statistics from '../components/Statistics.component';

configure({ adapter: new Adapter() });

// jest.mock('axios');

describe('<Statistics />', () => {
  test('should be selectable by class "girdStatistics"',() => {
    const wrapper = shallow(<Statistics />);

    expect(wrapper).toMatchSnapshot();
    wrapper.find('div').at(0).hasClass('girdStatistics');
    wrapper.find('div').at(0).hasClass('girdReact');
    wrapper.find('div').at(0).hasClass('girdVue');
    wrapper.find('div').at(0).hasClass('gridMovements');
  });
});
