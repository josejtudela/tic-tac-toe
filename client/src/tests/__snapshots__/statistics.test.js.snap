// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Statistics /> should be selectable by class "girdStatistics" 1`] = `
<Statistics>
  <div
    className="girdStatistics"
  >
    <div
      className="girdReact"
    >
      <h1>
        React wins:
        0
      </h1>
      <h3>
        Partidas:
      </h3>
    </div>
    <div
      className="girdVue"
    >
      <h1>
        Vue wins:
      </h1>
      <h3>
        Partidas:
      </h3>
    </div>
  </div>
</Statistics>
`;
